test/functional/tests/conftest.py

```diff
@@ -253,24 +253,21 @@ def base_prepare(item):
                 raid.stop()
                 for device in raid.array_devices:
                     Mdadm.zero_superblock(os.path.join('/dev', device.get_device_id()))
-                    Udev.settle()
 
         RamDisk.remove_all()
 
-        for disk in TestRun.dut.disks:
+        for disk in TestRun.disks.values():
             disk_serial = get_disk_serial_number(disk.path)
             if disk.serial_number != disk_serial:
-                raise Exception(
-                    f"Serial for {disk.path} doesn't match the one from the config."
-                    f"Serial from config {disk.serial_number}, actual serial {disk_serial}"
-                )
```

Comment:
After this change is disk serial still needed in dut config file?
Don't we want to have some type of info that serial from config does not match?

Reply:
It's not needed, but for compatibility's sake I left it in (no need to go and fix all your configs). IMO you shouldn't have to fill it if tests get it anyway. As for info that serials are different - what value do you expect from this info?